zephyr/wrapper.c

```diff
@@ -167,29 +167,8 @@ unsigned int _xtos_ints_off(unsigned int mask)
 	return 0;
 }
 
-/* Zephyr redefines log_message() and mtrace_printf() which leaves
- * totally empty the .static_log_entries ELF sections for the
- * sof-logger. This makes smex fail. Define at least one such section to
- * fix the build when sof-logger is not used.
- */
```

Comment:
should sof-logger `-l` support also be removed?

Reply:
not sure that's a good idea. As far as I can tell there's still some AMD (?) platforms and `imx8m` using XTOS. I don't have the toolchain right now to test but `smex` should still be included in those builds. AFAICT removing `-l` from the logger would break the only way of logging for those platforms (as you need the ldc file for it to work).